core/packages/PackageDB.h

```diff
@@ -63,6 +63,7 @@ class PackageDB final {
     const std::vector<std::string> &extraPackageFilesDirectorySlashDeprecatedPrefixes() const;
     const std::vector<std::string> &extraPackageFilesDirectorySlashPrefixes() const;
     const std::vector<std::string> &skipRBIExportEnforcementDirs() const;
+    const std::vector<core::NameRef> &layers() const;
```

Comment:
Let's maybe mention here that this the sort order matters in a doc comment?

e.g. let's document whether earlier layers are allowed to depend on later, or vice versa.